src/main/scala/com/databricks/labs/overwatch/utils/Tools.scala

```diff
@@ -881,5 +884,38 @@ object Helpers extends SparkSessionWrapper {
 
   }
 
+  def pipReport(etlDB: String = "overwatch_etl", detailed: Boolean = false):Dataset[Row] = {
+    try{
+      spark.catalog.getTable(s"${etlDB}.pipeline_report")
+      logger.log(Level.INFO, s"Overwatch has being deployed with  ${etlDB}")
+    }catch {
+      case e: Exception =>
+        val msg = s"Overwatch has not been deployed with  ${etlDB}"
+        logger.log(Level.ERROR, msg)
+        throw new BadConfigException(msg)
+    }
+
+    val pipReport = if (detailed) {
+      println("Report Extracted from pipeline_report")
+      table(s"${etlDB}.pipeline_report")
+    }else{
+      println("Report Extracted from pipReport")
+      table(s"${etlDB}.pipReport")
```

Comment:
select only the columns that are necessary and do it from pipeline_report so you don't have to join to get the packageVersion

organization_id, workspace_name, module_id, module_name, status, from_time, until_time, parsedConfig.packageVersion, Pipeline_SnapTS